website/docs/components/accordion/partials/code/component-api.md

```diff
@@ -0,0 +1,25 @@
+## Component API
+
+### Accordion
+
+The base Accordion component serves only as a wrapper to group together one or more AccordionItem child components.
+
+### AccordionItem
```

Comment:
Missing Component API table for `Accordion` + Changed content organization (see [ApplicationState](https://helios.hashicorp.design/components/application-state?tab=code#component-api) for example).
```suggestion

<Doc::ComponentApi as |C|>
  <C.Property @name="...attributes">
    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
  </C.Property>
</Doc::ComponentApi>

### Contextual Components

#### [A].AccordionItem
```